munit/src/main/scala/munit/MUnitRunner.scala

```diff
@@ -5,18 +5,17 @@ import org.junit.runner.notification.RunNotifier
 import java.lang.reflect.Modifier
 import org.junit.runner.notification.Failure
 import scala.util.control.NonFatal
-import org.junit.AssumptionViolatedException
 import org.junit.runner.manipulation.Filterable
 import org.junit.runner.manipulation.Filter
 import org.junit.runner.Runner
 import scala.collection.mutable
 
-class MUnitRunner(cls: Class[_ <: Suite]) extends Runner with Filterable {
+class MUnitRunner(val cls: Class[_ <: Suite]) extends Runner with Filterable {
```

Comment:
`cls` is references by public members, so it needs to be public. Dotty is quite strict about this and I think it makes sense.

I've also considered to make everything else private, but I got lazy.